index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <script src="https://kit.fontawesome.com/bf254b5930.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <div id="head-bar">
+    <span>My Logo</span>
+    <div class="menu-icon">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+  </div>
+
+  <main>
+    <header>
+      <h1>
+        I&rsquo;m Mariane<br>
+        Glad to see you!
+      </h1>
+      <p>
+        I&rsquo;m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don&rsquo;t hesitate to contact me.
+      </p>
```

Comment:
- [ ] You added your sub-title inside the paragraph tag. Kindly separate and style it as the Figma template design. Remember that you can use other levels like `<h2>` tags.